history/management/commands/pull_balance.py

```diff
@@ -41,15 +41,13 @@ def handle(self, *args, **options):
                     b = Balance(symbol=ticker,coin_balance=val,btc_balance=btc_val,exchange_to_btc_rate=exchange_rate_coin_to_btc,usd_balance=usd_val,exchange_to_usd_rate=exchange_rate_coin_to_btc,deposited_amount_btc=deposited_amount_btc if ticker =='BTC' else 0.00, deposited_amount_usd=deposited_amount_usd if ticker =='BTC' else 0.00)
                     b.save()
 
-        for b in Balance.objects.filter(date_str='0'):
-            # django timezone stuff , FML
-            b.date_str = datetime.datetime.strftime(b.created_on - datetime.timedelta(hours=int(7)),'%Y-%m-%d %H:%M')
+        for b in Balance.objects.filter(date_str=''):
```

Comment:
yes you are correct https://github.com/owocki/pytrader/blob/master/history/models.py#L207